apps/alarm/media-manager.js

```diff
@@ -37,13 +35,9 @@ function MediaManager (alarmCore) {
     if (!player) {
       return
     }
-    if (transient) {
-      player.pause()
-      this.audioFocus.resumeOnGain = true
-    } else {
-      this.stopMedia()
-      this.audioFocus.playUrl = null
-    }
+    this.stopMedia()
+    this.audioFocus.playUrl = null
+    this.alarmCore.clearAll()
```

Comment:
What would `this.alarmCore.clearAll()` do? I don't see the line in previous version.

Reply:
clear all alarm/reminder ringtone + tts + ringtone  timeout